functions.py

```diff
@@ -0,0 +1,191 @@
+import csv
+import datetime
+import exceptions_tests
+import os
+from pip._vendor.distlib.compat import raw_input
+
+EMPLOYEE_DOESNT_EXIST_MSG = "Sorry, the employee doesn't exist."
+EMPLOYEE_ALREADY_EXISTS_MSG = "Sorry, the employee is already exist."
+LATE_HOUR = "09:30:00"
+ADD_EMPLOYEES_FROM_FILE_MSG = "Please add the full file path of the employees you wish to add: "
+EMPLOYEE_ATTENDANCE_DOESNT_EXIST_MSG = "Sorry, the employee has no data in the attendance report."
+
+class Employee(object):
+
+    def __init__(self, employee_id, full_name, phone, age):
+        """
+        This function gets the employee's id, name, phone and age from user.
+        employee_id = a string.
+        full_name = a string.
+        phone = a string.
+        age = a string.
+        """
+        self.employee_id = employee_id
+        self.full_name = full_name
+        self.phone = phone
+        self.age = age
+
+
+class EmployeesList(object):
+
+    def __init__(self, list_of_employees):
+        """
+        This function gets employees file's path from user.
+        list_of_employees = a string.
+        """
+        self.list_of_employees = list_of_employees
+
+    def add_employee(self, new_employee):
+        """
+        This function gets the credentials of a new employee and employees file path from from user and adds it to the file.
+        new_employee = new employee's credentials.
+        """
+        with open(self.list_of_employees, 'r+') as list_of_employees_file:
+            list_of_employees_data = csv.reader(list_of_employees_file)
+            for row in list_of_employees_data:
+                if new_employee.employee_id in row:
+                    print(EMPLOYEE_ALREADY_EXISTS_MSG)
+                    return
+            list_of_employees_file.write(str(new_employee.employee_id) + ', ' + new_employee.name +
```

Comment:
instead of concate (+) use string foramtting 
:)

Reply:
done :)